spec/lib/miq_automation_engine/service_models/miq_ae_service_service_spec.rb

```diff
@@ -154,6 +154,7 @@ def invoke_ae
     end
 
     it "#finish_retirement" do
+      allow(Notification).to receive(:create)
```

Comment:
Is a notification only created sometimes?  If it is always created, these should use `expect`.